daemon/src/maker.rs

```diff
@@ -72,6 +68,48 @@ struct Opts {
     /// If enabled logs will be in json format
     #[clap(short, long)]
     json: bool,
+
+    #[clap(subcommand)]
+    network: Network,
```

Comment:
If there is no default, you might need to adapt the CI and want to update: `start_all.sh`